content/en/docs/tutorials/hello-minikube.md

```diff
@@ -142,6 +142,25 @@ recommended way to manage the creation and scaling of Pods.
     kubectl config view
     ```
 
+1. View application logs for a container in a pod.
+   
+   ```shell
+   # Return a snapshot of the logs from pod <pod-name>.
+   kubectl logs <pod-name>
+   ```
+
```

Comment:
In a tutorial, you don't have to explain the syntax and then provide an example.
An example is pretty straightforward.
If you want to point the users to the command syntax, you can add a pointer at
the end of the page.

Reply:
should I remove the explanation of syntax and example?

Reply:
It should be fine or desirable to remove line 147-151.